accessReviewScheduleDefinition docs refresh (#12447)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/accessreviewschedulesettings.md

## Properties
| Property    | Type   | Description |
| :---------------| :---------- | :---------- |

> | mailNotificationsEnabled|Boolean | Flag to indicate whether emails are enabled/disabled. Default value is `false`.               |

Reviewer: ```suggestion
| mailNotificationsEnabled|Boolean | Indicates whether emails are enabled or disabled. Default value is `false`.               |
```

> | reminderNotificationsEnabled|Boolean  | Flag to indicate whether reminders are enabled/disabled. Default value is `false`.  |

Reviewer: ```suggestion
| reminderNotificationsEnabled|Boolean  | Indicates whether reminders are enabled or disabled. Default value is `false`.  |
```

> | justificationRequiredOnApproval|Boolean | Flag to indicate whether reviewers are required to provide justification with their decision. Default value is `false`. |

Reviewer: ```suggestion
| justificationRequiredOnApproval|Boolean | Indicates whether reviewers are required to provide justification with their decision. Default value is `false`. |
```

> | defaultDecisionEnabled|Boolean | Flag to indicate whether default decision is enabled/disabled when reviewers do not respond. Default value is `false`. |

Reviewer: ```suggestion
| defaultDecisionEnabled|Boolean | Indicates whether the default decision is enabled or disabled when reviewers do not respond. Default value is `false`. |
```

| defaultDecision|String | Decision chosen if `defaultDecisionEnabled` is enabled. Can be one of "Approve", "Deny", or "Recommendation". |
| instanceDurationInDays|Int32 | Duration of each recurrence of review (`accessReviewInstance`) in number of days. |

> | recurrence|[patternedRecurrence](../resources/patternedrecurrence.md) | Detailed settings for recurrence using standard Outlook recurrence object. Note that only `weekly` and `absoluteMonthly` on **recurrencePattern** are supported. Use property **startDate** on **recurrenceRange** to determine the day the review will start on. |

Reviewer: ```suggestion
| recurrence|[patternedRecurrence](../resources/patternedrecurrence.md) | Detailed settings for recurrence using the standard Outlook recurrence object. Only `weekly` and `absoluteMonthly` on **recurrencePattern** are supported. Use the property **startDate** on **recurrenceRange** to determine the day the review starts. |
```

> | autoApplyDecisionsEnabled|Boolean | Flag to indicate whether auto-apply feature is enabled. When set to `false`, a user must apply the decisions manually once the reviewer completes the access review. When set to `true`, decisions are applied automatically once the reviewer completes the access review. Default value is `false`. |

Reviewer: ```suggestion
| autoApplyDecisionsEnabled|Boolean | Indicates whether the **auto-apply** feature is enabled. When set to `false`, a user must apply the decisions manually after the reviewer completes the access review. When set to `true`, decisions are applied automatically after the reviewer completes the access review. Default value is `false`. |
```
